e2e-tests/insurance/cypress/e2e/journeys/declarations/modern-slavery/modern-slavery.spec.js

```diff
@@ -0,0 +1,194 @@
+import { field as fieldSelector, headingCaption, yesRadio, noRadio } from '../../../../../../pages/shared';
+import { modernSlaveryPage } from '../../../../../../pages/insurance/declarations';
+import { PAGES } from '../../../../../../content-strings';
+import { DECLARATIONS_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance/declarations';
+import { FIELD_VALUES } from '../../../../../../constants';
+import { DECLARATIONS as DECLARATIONS_FIELD_IDS } from '../../../../../../constants/field-ids/insurance/declarations';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.DECLARATIONS.MODERN_SLAVERY;
+
+const {
+  ROOT: INSURANCE_ROOT,
+  DECLARATIONS: { MODERN_SLAVERY },
+} = INSURANCE_ROUTES;
+
+const {
+  MODERN_SLAVERY: { WILL_ADHERE_TO_ALL_REQUIREMENTS, HAS_NO_OFFENSES_OR_INVESTIGATIONS, IS_NOT_AWARE_OF_EXISTING_SLAVERY },
+} = DECLARATIONS_FIELD_IDS;
+
+const {
+  intro: { answerTheQuestions, guidingPrinciplesLink, ifYouSayNo },
+} = modernSlaveryPage;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Declarations - Modern slavery page - TODO EMS-4023', () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+      referenceNumber = refNumber;
+
+      url = `${baseUrl}${INSURANCE_ROOT}/${referenceNumber}${MODERN_SLAVERY}`;
```

Comment:
do we have a ticket to ensure we can't do this normally? as this section should not be unlocked yet

Reply:
Yes, EMS-4041. Otherwise this PR will be massive.

Also this is going into the `main-declarations-modern-slavery` branch. This will not be merged into `main` until everything is ready and we want to push it to production.